frontend/src/components/Scratch/ScratchPreferencesModal.tsx

```diff
@@ -22,6 +23,19 @@ function DiffPrefs() {
                 />
                 Automatically compile on change
             </label>
+            <label className={styles.intPreference}>
+                <input
+                    type="range"
```

Comment:
Consider disabling these inputs (and greying out this `label`) while "Automatically compile on changed" is unchecked

Reply:
Done. It now looks like this when disabled:
![image](https://user-images.githubusercontent.com/26349348/154254925-fdd1d9c3-dd5f-46b4-87ca-6b481fe19a8e.png)

enabled:
![image](https://user-images.githubusercontent.com/26349348/154254968-9ffa4135-e77c-40ea-8d93-2202f5047d93.png)